project_forecast_line_priority/data/ir_actions_server.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+    <record id="action_project_task_priority" model="ir.actions.server">
+        <field name="name">Project Task: Update forecast date end</field>
+        <field name="model_id" ref="project.model_project_task" />
+        <field name="binding_model_id" ref="project.model_project_task" />
+        <field name="binding_type">action</field>
+        <field name="state">code</field>
+        <field name="code">
```

Comment:
Create a method `_action_update_forecast_date_end`  in the model and make the code field a single call to this method.

This will allow the method to be overridden in other modules and also will make the code available for unit test and the code will be subjected to linting etc. So in general code in a cronjob or in a server action should preferably always be a single line, calling a function in the model (that is for cronjobs or server actions added through code).